extensions/emmet/src/test/partialParsingStylesheet.test.ts

```diff
@@ -60,7 +60,7 @@ p {
 /* .foo { op.3
 dn	{
 */
-	@
+	bgc
```

Comment:
@ is now allowed outside of declaration blocks now, so we change this to an abbreviation that shouldn't be allowed outside of a declaration block, to preserve the older behaviour.